docs/architecture/governance.md

```diff
@@ -1,50 +1,87 @@
 ---
-title: Governance
+title: Governance and self-amendment
 last_update:
-  date: 3 July 2023
+  date: 22 February 2024
 ---
 
-## What is Self Amendment?
+Tezos incorporates a built-in, on-chain mechanism for proposing, selecting, testing, and activating protocol upgrades without the need to hard fork.
+This mechanism makes Tezos a self-amending blockchain and allows any user to propose changes.
```

Comment:
Note that I added the "economic" qualifier to avoid any possible confusion with other protocols in Tezos: consensus protocol, peer-to-peer protocol, RPC protocol, etc.

BTW, the page <https://docs-staging-git-voting-trili-tech.vercel.app/architecture/rpc> should be renamed to any of "RPC interface", "RPC API", "RPCs", "Tezos RPCs" or such, because it is not about the RPC protocol itself but about a set of of RPC *calls*.

Reply:
Making that update here: https://github.com/trilitech/tezos-developer-docs/pull/341

Reply:
thanks, approved